next.config.js

```diff
@@ -24,32 +26,41 @@ const nextConfig = {
     return [
       {
         source: '/instructors/:instructor',
-        destination: '/s/content-by-:instructor',
+        destination: `${searchUrlRoot}/content-by-:instructor`,
         permanent: true,
       },
       {
         source: '/podcasts',
-        destination: '/s?type=podcast',
+        destination: `${searchUrlRoot}?type=podcast`,
         permanent: true,
       },
       {
         source: '/talks',
-        destination: '/s?type=talk',
+        destination: `${searchUrlRoot}?type=talk`,
         permanent: true,
       },
       {
         source: '/courses',
-        destination: '/s?type=course',
+        destination: `${searchUrlRoot}?type=course`,
+        permanent: true,
+      },
+      {
+        source: '/lessons',
+        destination: `${searchUrlRoot}?type=lesson`,
         permanent: true,
       },
-      {source: '/lessons', destination: '/s?type=lesson', permanent: true},
-      {source: '/search', destination: '/s', permanent: true},
-      {source: '/instructors', destination: '/s', permanent: true},
-      {source: '/browse', destination: '/s', permanent: true},
-      {source: '/browse/:context', destination: '/s', permanent: true},
+      {source: '/search', destination: searchUrlRoot, permanent: true},
+      {source: '/instructors', destination: searchUrlRoot, permanent: true},
+      {source: '/browse', destination: searchUrlRoot, permanent: true},
+      {source: '/browse/:context', destination: searchUrlRoot, permanent: true},
       {
         source: '/browse/:context/:tag',
-        destination: '/s/:tag',
+        destination: `${searchUrlRoot}/:tag`,
+        permanent: true,
+      },
+      {
+        source: '/s/:all*',
+        destination: 'https://app.egghead.io/s/:all*',
```

Comment:
Since this is redirecting to `app.egghead.io`, should `src/pages/s/[[...all]].tsx` be deleted? https://github.com/eggheadio/egghead-next/blob/jh/change-search-root/src/pages/s/%5B%5B...all%5D%5D.tsx

Reply:
forgot to `git add .` so it didn't register when I renamed!